### PR TITLE
Use tags to distinguish different apps in Sentry

### PR DESCRIPTION
**Pull request by djmitche**
Not actually tested with Sentry itself, but this seems to pass CI tests.

### CODE REVIEW

**Comment by djmitche** [comment]
Fixes #988.

**Review by jwhitlock — CHANGES_REQUESTED** [review]
Thanks @djmitche! I was able to test this with the stage DSN,

```
make shell
SENTRY_DSN=https://<etc> ichnaea/scripts/sentry_test.py clitest
```

In Sentry, the tags are:

* level: error
* node: sentry_test
* release: unknown
* server_name: 59e3c4bbdec4

I think I'd prefer `app` or `app_type` to `node`, but it is not a strong preference.

Can you run ``make lintfix`` to get black happy, and rebase on the current main branch?

File: ichnaea/log.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def configure_logging():
 }
 
 
-def configure_raven(transport=None, _client=None):
+def configure_raven(transport=None, tags={}, _client=None):
```
[the remark this answers is not in the document]
Comment (djmitche) [reply ichnaea/log.py:136]: Yeah, good point.  My python's a little rusty (heh) and I forgot about that issue.  And I will eventually learn to remember lint!

**Review by jwhitlock — APPROVED** [review]
Tested as before, now app=sentry_test. Thanks @djmitche, looks good!

**Pull request merged**